app.js

```diff
@@ -1,13 +1,7 @@
+import config from "./config";
+
 (function() {
   // Initialize Firebase
```

Comment:
This doesn't need to be secured according to firebase but yes its better to isolate it out.

Reply:
Yes it does not but it is better to not to have API keys in the repo

Reply:
since others who work on the fork will use their own project